css/form-display.css

```diff
@@ -0,0 +1,47 @@
+@media print {
+    .noprint {
+       visibility: hidden;
+    }
+    @page {
+        size: auto;   /* auto is the initial value */
+        margin: 0;  /* this affects the margin in the printer settings */
+    }
+ }
+
+footer {
+    background-color: #ffffff; 
+    opacity: 100%;
+    width: 189mm; /* Match .styBodyClass width to make this appear on the bottom of the form*/
+    display: flex; 
+    position: fixed; 
+    bottom: 0; 
+    left: 0;
+    align-items: center;
+    margin-left: 0;
+    height: 5em;
+}
+
+button {
+    font-family: "Lato","Helvetica Neue", Helvetica,Arial,sans-serif;
+    font-size: 1rem;
+    font-weight: 700;
+    color: #ffffff;
+    background-color: #1F0C7B;
+    transition: opacity 0.3s ease-out, outline 0.3s ease-out;
+    border: none;
+    border-radius: .5em;
+    cursor: pointer;
+    position: relative;
+    left: 45%;
+}
```

Comment:
small thing -- I'm wondering if it would be worth it to add a class to this button and put the styling on that class, since this styling contains `left: 45%` which would be specific to this one button? Just in case another button were to be added someday (maybe "save to PDF" or something?).